code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -104,6 +104,11 @@
 	QDEL_IN(src, 10 SECONDS)
 	return ..()
 
+/mob/living/simple_animal/hostile/abnormality/mountain/gib()
+	if(health > 0 || phase > 1)
+		return
```

Comment:
Can we make it so that this throws him down a level? The actual visible gib effect being large chunks of bodies sloshing off him as he gets smaller. I think it'd be a cool way of doing it without just removing the previous strategy.

Reply:
Done